src/thunderbots_msgs/msg/PlayInfo.msg

```diff
@@ -0,0 +1,8 @@
+# This message contains information about the current Play and Tactics our AI is running
+
+# the name of this team
+string play_type
+
+string play_name
+
+string[] tactics
```

Comment:
could we change to `robot_tactic_assignment`?

Reply:
renamed